pages/07-solid-gauge-chart/simple.page.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This is a variation of the core chart, as we don't have a special component for gauge. Let's move the page to under 03-core folder.